shipping-rates/template/src/dashboard/pages/page.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useCallback, useEffect, useState } from "react";
+import {
+  Box,
+  Breadcrumbs,
+  Button,
+  Cell,
+  Layout,
+  Loader,
+  Page,
+} from "@wix/design-system";
+import { dashboard } from "@wix/dashboard";
+import { ShippingCostsForm } from "../components/ShippingDeliveryMethodForm";
+import { ShippingAppData, ShippingCosts } from "../../types/types";
+import { RecentOrdersCard } from "../components/RecentOrdersCard";
+import { ACTIVATE_PLUGIN_MODAL_ID, WixPageId } from "../../consts";
+import { useShippingAppData } from "../hooks/use-shipping-app-data";
+import { withProviders } from "../withProviders";
+import { UpgradeCard } from "../components/UpgradeCard";
+import FreeTrialMessage from "../components/FreeTrialMessage";
+import { appInstances } from "@wix/app-management";
+import "@wix/design-system/styles.global.css";
+
+function ShippingRatesPage() {
+  const { showToast, navigate } = dashboard;
+  const { getShippingAppData, setShippingAppData } = useShippingAppData();
+
+  const [appInstance, setAppInstance] = useState<appInstances.AppInstance>();
+  const [currentShippingAppData, setCurrentShippingAppData] = useState<
+    ShippingAppData | undefined
+  >(getShippingAppData.data);
+
+  useEffect(() => {
+    async function getAppInstance() {
+      const { instance } = await appInstances.getAppInstance();
+      setAppInstance(instance);
+    }
+
+    getAppInstance();
+
+    setCurrentShippingAppData(getShippingAppData.data);
+  }, [getShippingAppData.data]);
+
+  const onSave = useCallback(async () => {
+    if (!currentShippingAppData) return;
+
+    try {
+      await setShippingAppData.mutateAsync(currentShippingAppData);
+
+      showToast({
+        message: "Shipping rates saved successfully.",
+        type: "success",
+      });
+    } catch (error) {
+      showToast({
+        message: "Failed to save shipping rates.",
+        type: "error",
+      });
+    }
+  }, [setShippingAppData.mutateAsync, currentShippingAppData, showToast]);
+
+  const setCostsForMethod = useCallback(
+    (shippingMethodCode: string) => (costs: ShippingCosts) => {
+      setCurrentShippingAppData({
+        ...currentShippingAppData,
+        shippingMethods: currentShippingAppData!.shippingMethods.map(
+          (shippingMethod) =>
+            shippingMethod.code === shippingMethodCode
+              ? { ...shippingMethod, costs }
+              : shippingMethod
+        ),
+      });
+    },
+    [currentShippingAppData]
+  );
+
+  const shouldShowUpgradeCard = appInstance && appInstance.isFree;
+
+  return (
+    <Page height="100vh">
```

Comment:
```suggestion
    <Page>
```
fixed by [this](https://github.com/wix-private/wix-cli/pull/2867)

Reply:
Yea maybe you're not on the latest CLI version. Tried without it and it looked ok to me (if it does not for lets check it together)